deployment/clouddeploy/gke-workers/environments/oss-vdb-test/importer-deleter.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: importer-deleter
+spec:
+  schedule: "* */6 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          tolerations:
+          - key: workloadType
+            operator: Equal
+            value: importer-pool
+          containers:
+          - name: importer-deleter
+            env:
+            - name: GOOGLE_CLOUD_PROJECT
+              value: oss-vdb-test
+            image: importer
+            args: ["--delete", "--deletion_threshold=10", "--public_log_bucket=osv-test-public-import-logs"]
+            imagePullPolicy: Always
+            volumeMounts:
+              - mountPath: "/work"
+                name: "ssd"
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                cpu: 1
+                memory: "1G"
+              limits:
+                cpu: 1
+                memory: "2G"
+          nodeSelector:
+            cloud.google.com/gke-nodepool: importer-pool
```

Comment:
I think this `importer-pool` has only one node, which iirc was done so that the main importer could have some storage persist.

Could @another-rex confirm if this might be a problem?

Reply:
Yes, it only has one node with only 2 CPU, which means the deleter and the importer can't be scheduled at the same time.

Reply:
Removed `nodeSelector` in [6f1885e](https://github.com/google/osv.dev/pull/2133/commits/6f1885e6e814b2a7aea43c3bb6bb37c490a62816)

Reply:
Could you PTAL as of [5caf9c1](https://github.com/google/osv.dev/pull/2133/commits/5caf9c15a93c3037accffcafe23fba9af057b677)?